torch/_dynamo/eval_frame.py

```diff
@@ -737,7 +738,15 @@ def __init__(
                 # Fill node.mata["val"] with faketensor from the input,
                 # if it's not found in matched_input_elements_positions
                 if fake_mode is not None and isinstance(flat_args[i], torch.Tensor):
-                    arg.node.meta["val"] = fake_mode.from_tensor(flat_args[i])
+                    arg.node.meta["val"] = fake_mode.from_tensor(
+                        flat_args[i],
+                        dynamic_dims=[
+                            DimDynamic.DYNAMIC
+                            if d in flat_dynamic_dims[i]
+                            else DimDynamic.STATIC
+                            for d in range(len(flat_args[i].shape))
+                        ],
+                    )
```

Comment:
Hmm, is this the correct place to update the inputs vs. doing this in export or something? 
cc @avikchaudhuri

Reply:
At least this is the place what @tugsbayasgalan told me... 

The problem is these inputs are not tracked in dynamo, therefore you couldn't find a place to introduce these input naturally in the graph. If you look at the blame, these lines are maintained by export for quite a while.